numba/tests/test_mviewbuf.py

```diff
@@ -0,0 +1,24 @@
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+from numba import unittest_support as unittest
+from numba import mviewbuf
+
+class TestMviewBuf(unittest.TestCase):
+    def test_buffer_proxy(self):
+        dt = np.dtype([('a', np.int32), ('b', np.float32)])
+        rec = np.recarray(1, dt)[0]
+        bp = mviewbuf.BufferProxy(rec)
+        ptr = mviewbuf.memoryview_get_buffer(bp)
+
+        # Check that the returned buffer is a nonzero integer
```

Comment:
I think it might be a long under Python  2 (especially under 64-bit Windows, which has 64-bit pointers but 32-bit C "longs").